src/encoded/static/components/inputs/object.js

```diff
@@ -20,27 +21,30 @@ var openLinksInNewWindow = function(e) {
 };
 
 
-var SearchBlockEdit = React.createClass({
+var SearchBlockEdit = createReactClass({
     render: function() {
         var styles = {maxHeight: 300, overflow: 'scroll', clear: 'both' };
         return (
-            <div className="well" style={styles} onClick={openLinksInNewWindow}>
+            <div
+                className="well" style={styles} onClick={openLinksInNewWindow}
+                ref={(comp) => { this.domNode = comp; }}
```

Comment:
`this.getDOMNode()` isn’t a thing in React 15. You have to use a `ref` callback function to assign the rendered DOM node to an object property that you then use once the component mounts.